hw_26_paparazzi/js/script.js

```diff
@@ -0,0 +1,13 @@
+function structureUserInfo(userName) {
+    let userInfo = {name: userName};
+    return function (userRole) {
+        userInfo.role = userRole;
+        return userInfo;
+    }
+}
```

Comment:
That's right. However you could beautify this 

```suggestion
function structureUserInfo(name) {
    return function (role) {
      return { name, role }
    }
}
```